tools/serverpod_cli/lib/src/util/pubspec_helpers.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:io';
+
+import 'package:pubspec_parse/pubspec_parse.dart';
+
+Pubspec? tryParsePubspec(File pubspecFile) {
+  try {
+    var yaml = pubspecFile.readAsStringSync();
+    var pubspec = Pubspec.parse(yaml);
+    return pubspec;
+  } catch (e) {
+    print('Error while parsing pubspec file: ${pubspecFile.path}');
```

Comment:
I wouldn't expect a tryParsePubspec to print things as a side effect. I think it would be better to catch the error at a higher level.